manager/integration/tests/test_basic.py

```diff
@@ -5021,8 +5023,7 @@ def test_backup_volume_restore_with_access_mode(core_api, # NOQA
     assert volume_sp_access_mode.accessMode == overridden_restored_access_mode
 
 
-@pytest.mark.skip(reason="TODO")  # NOQA
-def test_delete_backup_during_restoring_volume():  # NOQA
+def test_delete_backup_during_restoring_volume(set_random_backupstore, client, core_api, volume_name):  # NOQA
```

Comment:
The `core_api` and `volume_name` seems redundant.

If want to keep the volume_name you can use `vol_v1_name = volume_name + "-v1"`.

Reply:
> The `core_api` and `volume_name` seems redundant.
> 
> If want to keep the volume_name you can use `vol_v1_name = volume_name + "-v1"`.

Remove the `core_api` and `volume_name` and the test result is passed.
https://ci.longhorn.io/job/private/job/longhorn-tests-regression/2982/
![Screenshot_20230210_101921](https://user-images.githubusercontent.com/104337648/217983814-bf22f945-3d48-4524-b54f-160b853e54eb.png)